Source/engine/assets.cpp

```diff
@@ -43,8 +43,9 @@ SDL_RWops *OpenAsset(const char *filename, bool threadsafe)
 
 	// Files next to the MPQ archives override MPQ contents.
 	SDL_RWops *rwops;
-	if (paths::MpqDir()) {
-		const std::string path = *paths::MpqDir() + relativePath;
+	const auto &searchPaths = paths::GetMPQSearchPaths();
+	for (const auto &mpqDir : searchPaths) {

```

Comment:
This calls `OpenMpqFile(filename, ...)` multiple times with the same arguments.

Reply:
Could you elaborate?
The `OpenMpqFile` call is outside the `searchPaths` loop, so it should only be called once (as in master).

Reply:
Oh, sorry, misread the code. Nevermind